src/main/java/org/sefglobal/scholarx/controller/UserController.java

```diff
@@ -0,0 +1,41 @@
+package org.sefglobal.scholarx.controller;
+
+import org.sefglobal.scholarx.exception.ResourceNotFoundException;
+import org.sefglobal.scholarx.model.Mentee;
+import org.sefglobal.scholarx.model.Mentor;
+import org.sefglobal.scholarx.service.MenteeService;
+import org.sefglobal.scholarx.service.MentorService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
```

Comment:
No need of a new class add 
1. Mentor services to existing `ProgramController`
2. Mentee services to `MentorController`

Reply:
Done